src/components/GameView/GameOverDialog.vue

```diff
@@ -46,6 +72,14 @@ export default {
         // do nothing - parent controls whether dialog is open
       },
     },
+    ...mapState({
+      playingFromDeck: ({ game }) => game.playingFromDeck,
+      game: ({ game }) => game,
```

Comment:
Mapping the game this way feels weird but I get why you're doing it. It might be better to eventually convert these to getters, nothing that will hold up this PR though. I think we currently just use a computed prop in each component that references the game state as-is which is already weird, too. cc @itsalaidbacklife

Reply:
huh, yeah. Likely worth revisiting this holistically but it'll change when we move to pinia so let's wait until then to dig in. Imo this is fine but agreed it's slightly wonky as is having a separate computed prop

Reply:
Yep, that works. Just calling it out so we don’t lose it in the shuffle.